TESTING-INSTRUCTIONS.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+### Try: Use the store timezone to make time data requests #6632
+
+1. Create a store with a timezone significantly ahead of or behind the timezone you currently reside in.
```

Comment:
Maybe make these a little more specific, by mentioning you can do this by going to **Settings > General** Timezone